tests/test_dbus.py

```diff
@@ -40,6 +40,12 @@ def test_get_login_manager_proxy():
     assert login_manager.CanPowerOff() in {("yes",), ("challenge",)}
 
 
+def get_service_manager_proxy():
```

Comment:
> I removed the commented lines but left the rest in assumption that it covers parial testing.

If you want to use that function as a test, you need to prefix `test_`:
```suggestion
def test_get_service_manager_proxy():
```